config/test.exs

```diff
@@ -20,4 +20,9 @@ config :logger, level: :warn
 config :elixir_auth_google,
   client_id: "631770888008-6n0oruvsm16kbkqg6u76p5cv5kfkcekt.apps.googleusercontent.com",
   client_secret: "MHxv6-RGF5nheXnxh1b0LNDq",
+  httpoison_mock: true
+
+config :auth_plug,
+  api_key: System.get_env("AUTH_API_KEY"),
+    # "2PzB7PPnpuLsbWmWtXpGyI+kfSQSQ1zUW2Atz/+8PdZuSEJzHgzGnJWV35nTKRwx/authdev.herokuapp.com",
```

Comment:
I was hard-coding the `api_key` and then `AuthPlug.Token.client_id/0` is reading it from there instead of `.env` ...
So I had created a misdirection and had to do a bunch of `IO.inspect`-ing to figure it out. 🙄